src/components/Forms/CreateAppForm.tsx

```diff
@@ -110,6 +114,41 @@ export const CreateSingleAppForm = (props: {
     setEnvVariables([...envVariables, { key: "", value: "" }]);
   };
 
+  const parseVariablesFromContent = (content: string) => {
+    const lines = content.split("\n");
+    return lines
+      .map((line) => {
+        const [key, ...values] = line.split("=");
+        // eslint-disable-next-line curly
+        if (!key || values.length === 0) return null;
```

Comment:
you should always let the code linter show you how to fix such, or you also run `yarn run prettier:write` in your terminal, instead of adding these eslint disable comments.